tee-worker/cli/src/base_cli/mod.rs

```diff
@@ -103,7 +97,6 @@ impl BaseCommand {
 			BaseCommand::RegisterTcbInfo(cmd) => cmd.run(cli),
 			// Litentry's commands below
 			BaseCommand::PrintSgxMetadataRaw => print_sgx_metadata_raw(cli),
-			BaseCommand::LinkIdentity(cmd) => cmd.run(cli),
```

Comment:
What about `PrintSgxMetadataRaw` (just the line above)?  Do we have help message for it?

Reply:
Yes, it has help message, though it's a bit verbose 🤔 
![image](https://github.com/litentry/litentry-parachain/assets/2113554/9f866eee-4ff6-4b99-9107-6f645b2c760b)